src/main/java/com/algorand/algosdk/builder/transaction/AssetAcceptTransactionBuilder.java

```diff
@@ -0,0 +1,97 @@
+package com.algorand.algosdk.builder.transaction;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.transaction.Transaction;
+
+import java.math.BigInteger;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * Build an asset accept transaction.
+ */
+public class AssetAcceptTransactionBuilder<T extends AssetAcceptTransactionBuilder<T>> extends TransactionBuilder<T> {
+    protected Address acceptingAccount = null;
+    protected BigInteger assetIndex = null;
+
+    /**
+     * Initialize a {@link AssetAcceptTransactionBuilder}.
+     */
+    public static AssetAcceptTransactionBuilder<?> Builder() {
+        return new AssetAcceptTransactionBuilder<>();
+    }
+
+    protected AssetAcceptTransactionBuilder() {
+        super(Transaction.Type.AssetTransfer);
+    }
+
+    @Override
+    protected Transaction buildInternal() {
+        if (sender != null) {
+            throw new IllegalArgumentException("Do not use 'sender' for asset transfer transactions. Only use 'assetSender'");
+        }
+
+        return Transaction.createAssetAcceptTransaction(
+                acceptingAccount,
+                fee,
+                firstValid,
+                lastValid,
+                note,
+                genesisID,
+                genesisHash,
+                assetIndex);
+    }
+
+    /**
+     * Set the acceptingAccount.
+     */
+    public T acceptingAccount(Address acceptingAccount) {
```

Comment:
Please copy over the comments here. 
e.g.
acceptingAccount is a checksummed, human-readable address that will accept receiving the asset.